src/storage.js

```diff
@@ -0,0 +1,59 @@
+import {getCookie, setCookie} from "./cookies.js";
+
+function EvolvStorageManager(useCookies, allowPersistence) {
```

Comment:
JSDoc comments for good measure? Especially since `useCookies` is not expecting a boolean as the variable name would suggest.